src/containers/DevSite.jsx

```diff
@@ -111,18 +111,27 @@ class Devsite extends Component {
     );
   }
 
+  findRouteCategory(meta, currRoute, homeRoute) {
+    const shortRoute = currRoute.slice(1);
+    for (let i = 0; i < meta.length; i += 1) {
+      const { route, category } = meta[i];
+      if (shortRoute === route) {
+        return category;
+      } else if (route === homeRoute &&
+        (currRoute === '/' || currRoute === '/home')) {
+        return category;
+      }
```

Comment:
Both the `if` and the `else if` stanzas return the same value if true. Would you consider joining them into a single if statement?

Reply:
Yea, I just wanted to avoid a super long `if` chain. I don't mind changing it.